.github/pull_request_template.md

```diff
@@ -19,6 +19,16 @@ Bad example:
 change ingress
 -->
 
+### Risk Assessment
+
+<!-- Describe the risk of this change and how it will be tested, deployed, and verified. -->
```

Comment:
```suggestion
<!-- WIP: Some teams are experimenting with this section. -->
<!-- Describe the risk of this change and how it will be tested, deployed, and verified. -->
```

Reply:
Suggested a WIP note to help with change management.

Reply:
Thanks. Added a bit to it:

> WIP: Some teams are experimenting with this section. Feel free to remove.